lms/static/js/dashboard/views/course_view.js

```diff
@@ -0,0 +1,161 @@
+;(function (define, accessibleModal) {
+    'use strict';
+
+    define(['jquery', 'backbone', 'underscore', 'underscore.string',
+            'text!templates/dashboard/course.underscore'],
+        function ($, Backbone, _, _s, courseTemplate) {
+
+            if (_.isUndefined(_s)) {
+                _s = _.str;
+            }
+
+            var CourseView = Backbone.View.extend({
+                el: '<li class="course-item" />',
+                events: {
+                    'click .action-more': 'toggleCourseActionsDropDown',
+                    'click .action-unenroll': 'unEnroll',
+                    'click .action-email-settings': 'emailSettings',
+                    'click #upgrade-to-verified': 'upgradeToVerified',
+                    'click #block-course-msg a[rel="leanModal"]': 'unRegisterBlockCourse'
+                },
+
+                initialize: function (options) {
+                    this.template = _.template(courseTemplate);
+                    this.listenTo(options.tabbedView, 'rendered', this.rendered);
+                    this.settingsModel = options.settingsModel;
+
+                    /* Mix non-conflicting functions from underscore.string
+                     * (all but include, contains, and reverse) into the
+                     * Underscore namespace
+                     */
+                    _.mixin(_s.exports());
+                },
+
+                render: function () {
+                    this.$el.html(this.template({model: this.model.attributes, settings: this.settingsModel}));
+                    return this;
+                },
+
+                rendered: function () {
+                    var $actionUnroll = this.$('.action-unenroll'),
+                        $unRegisterBlockCourse = this.$('#unregister_block_course'),
+                        $actionEmailSettings = this.$('.action-email-settings'),
+                        modalType = {
+                            unenroll: 'unenroll',
+                            emailSettings: 'email-settings'
+                        };
+
+                    this.bindModal($actionUnroll, modalType.unenroll);
+                    this.bindModal($unRegisterBlockCourse, modalType.unenroll);
+                    this.bindModal($actionEmailSettings, modalType.emailSettings);
+                },
+
+                bindModal: function ($selector, type) {
+                    var trigger,
+                        id = _.uniqueId('unenroll-');
+
+                    $selector.leanModal({
+                        overlay: 1,
+                        closeButton: ".close-modal"
```

Comment:
Please be consistent with quotation marks (single should be used here because they are in the rest of the file)